src/lib.rs

```diff
@@ -324,10 +327,10 @@ impl LightningNode {
         todo!()
     }
 
-    // TODO remove unused_variables after breez sdk implementation
-    #[allow(unused_variables)]
-    pub fn get_payment_max_routing_fee_mode(&self, amount_sat: u64) -> MaxRoutingFeeMode {
-        todo!()
+    pub fn get_payment_max_routing_fee_mode(&self, _amount_sat: u64) -> MaxRoutingFeeMode {
+        MaxRoutingFeeMode::Relative {
```

Comment:
How would you suggest accommodating for this?

As far as I understand, if the fee is smaller than `exemptfee` we ignore the `maxfeepercent`. But in this method we do not know about the fees. As written in the Jira Issue it might make sense to deprecate this method.

Reply:
Okay you're right. Why deprecating it, well couldn't we just let the node handle this for us or are we exposing this due to UX benefits?